companies/eastbankclub.json

```diff
@@ -0,0 +1,23 @@
+{
+    "slug": "eastbankclub",
+    "relevant-countries": [
+        "de"
+    ],
+    "categories": [
+        "health"
+    ],
+    "name": "East Bank Club The Fitness Factory GmbH",
+    "runs": [
+        "SUPERFIT"
+    ],
+    "address": "Postfach 410108\n12111 Berlin\nDeutschland",
+    "phone": "+49 30 688381768",
+    "fax": "+49 30 688381750",
+    "email": "member@superfit.club",
+    "web": "https://superfit.club",
+    "sources": [
+        "https://superfit.club/datenschutz/",
+        "https://superfit.club/impressum/"
+    ],
+    "quality": "tested"
```

Comment:
This is new, so you couldn't have know but we now require `required-elements` for tested records (see #811). If you happen to know the elements required to send a request to this company, it would be great if you could add them. Otherwise, we can just downgrade to `verified`.

Reply:
I can add the fields (name, address, DOB) that I used in my request but I don't know if that's the minimal set...

Reply:
Besides: I tested the GDPR request via fax. Does this conflict with `suggested-transport-medium` = `email`?

Reply:
> I can add the fields (name, address, DOB) that I used in my request but I don't know if that's the minimal set...

From my experience, specifying the DOB is often not necessary, not sure about this company, though.

> Besides: I tested the GDPR request via fax. Does this conflict with suggested-transport-medium = email?

In this case, I think we should just start with `quality: tested`. Should you ever send a request to them again, it would be great if you could then add the required elements and upgrade the record to `verified`.

Reply:
> In this case, I think we should just start with quality: tested. Should you ever send a request to them again, it would be great if you could then add the required elements and upgrade the record to verified.
I suppose you mean the other way round: now `verified`, later `tested`?

Reply:
Oh yes. Sorry.

Reply:
done